champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/types/CooldownSkill.java

```diff
@@ -18,12 +17,7 @@ default boolean isCancellable(){
     }
 
     default boolean shouldDisplayActionBar(Gamer gamer) {
-        final Player found = gamer.getPlayer();
-        if (found == null || !hasSkill(found)) {
-            return false;
-        }
-
-        return SkillWeapons.getTypeFrom(found.getInventory().getItemInMainHand()) == getType();
+        return isHolding(gamer.getPlayer()) && (getType() == SkillType.AXE || getType() == SkillType.SWORD);
```

Comment:
Is this also the cool down action bar? What about drop item skills such as smoke bomb

Reply:
We had those on but they were conflicting with other skill's action bar, I think that's also the reason Mineplex removed it.